BlockchainSdk/BlockchainService/BlockBook/BlockBookUtxoProvider.swift

```diff
@@ -29,61 +29,91 @@ class BlockBookUtxoProvider {
         self.provider = NetworkProvider<BlockBookTarget>(configuration: networkConfiguration)
     }
     
-    func addressData(address: String, parameters: BlockBookTarget.AddressRequestParameters) -> AnyPublisher<BlockBookAddressResponse, Error> {
-        provider
-            .requestPublisher(target(for: .address(address: address, parameters: parameters)))
-            .filterSuccessfulStatusAndRedirectCodes()
-            .map(BlockBookAddressResponse.self)
-            .eraseError()
-            .eraseToAnyPublisher()
+    func addressData(
+        address: String,
+        parameters: BlockBookTarget.AddressRequestParameters
+    ) -> AnyPublisher<BlockBookAddressResponse, Error> {
+        executeRequest(.address(address: address, parameters: parameters))
     }
     
     func unspentTxData(address: String) -> AnyPublisher<[BlockBookUnspentTxResponse], Error> {
-        provider
-            .requestPublisher(target(for: .utxo(address: address)))
-            .filterSuccessfulStatusAndRedirectCodes()
-            .map([BlockBookUnspentTxResponse].self)
-            .eraseError()
-            .eraseToAnyPublisher()
+        executeRequest(.utxo(address: address))
     }
     
     func getFeeRatePerByte(for confirmationBlocks: Int) -> AnyPublisher<Decimal, Error> {
-        provider
-            .requestPublisher(target(for: .fees(confirmationBlocks: confirmationBlocks)))
-            .filterSuccessfulStatusAndRedirectCodes()
-            .map(BlockBookFeeResponse.self)
-            .tryMap { [weak self] response in
-                guard let self else {
-                    throw WalletError.empty
-                }
-                
-                if response.result.feerate <= 0 {
+        executeRequest(
+            .fees(NodeRequest.estimateFeeRequest(confirmationBlocks: confirmationBlocks)),
+            responseType: BlockBookFeeResponse.self
+        )
+        .tryMap { [weak self] response in
```

Comment:
а вот так соберется? Хочется только один `executeRequest` иметь

```suggestion
        executeRequest(
            .fees(NodeRequest.estimateFeeRequest(confirmationBlocks: confirmationBlocks))
        )
        .tryMap { [weak self] (response: BlockBookFeeResponse) in
```

Reply:
компилятор не резолвит без явного типа, выше где-то скрин кидал